libs/openFrameworks/types/ofBaseTypes.h

```diff
@@ -527,42 +547,51 @@ class ofBaseVideoPlayer: virtual public ofBaseVideo{
 	virtual void 				setPaused(bool bPause);
 	/// \brief Set the position of the playhead.
 	///
-	/// The value used to set the playhead position should be a normalized float
-	/// between 0.0 and 1.0 that represents percentage of video completion.
+	/// This value is a normalized floating point value between 0.0 and 1.0 that
+	/// represents the position of the playhead. 0.0 maps to the first frame of
+	/// the loaded video and 1.0 maps to the last frame of the loaded video.
 	///
-	/// \param pct A normalized percentage value between 0.0 and 1.0.
+	/// \param pct A value between 0.0 and 1.0 representing playhead position.
 	virtual void 				setPosition(float pct);
 	/// \brief Set the volume of the video player.
 	///
-	/// The value used to set the video player volume should be a normalized
-	/// float between 0.0 and 1.0 that represents the desired volume/gain.
+	/// This value is a normalized floating point value between 0.0 and 1.0 that
+	/// represents the video player volume. 0.0 maps to silence and 1.0 maps to
+	/// maximum volume.
 	///
-	/// \param volume A normalized volume value between 0.0 and 1.0.
+	/// \param volume A value between 0.0 and 1.0 representing volume.
 	virtual void 				setVolume(float volume);
 	/// \brief Set the video loop state.
-	///
-	/// Loop states include OF_LOOP_NONE, OF_LOOP_PALINDROME, amd OF_LOOP_NORMAL
-	///
 	/// \param state The loop state of the video.
 	/// \sa ::ofLoopType
 	virtual void 				setLoopState(ofLoopType state);
 	/// \brief Set the video playback speed.
 	///
-	/// The speed parameter supplied to setSpeed() considers 1.0 to be regular
-	/// playback speed, 0.5 to be half speed, and 2.0 to be double speed, etc...
+	/// When the loop state is OF_LOOP_NONE or OF_LOOP_NORMAL, positive speed
+	/// will scale a forward playback rate while a negative speed will scale a
+	/// a backward playback rate. When the loop state is OF_LOOP_PALINDROME,
+	/// the direction of playback will change each loop, but the playback rate
+	/// will still be scaled by the absolute value of the speed.
+	///
+	/// To play a video forward at normal speed, set the loop state to
+	/// OF_LOOP_NONE or OF_LOOP_NORMAL and a speed of 1.0. To double the
+	/// playback rate, set the speed to 2.0. To play a video backward, set the
+	/// speed to a negative number. A speed 0.25 will play the video at 1/4 the
+	/// the normal rate and a rate of 0.0 will effectively stop playback.
```

Comment:
@brannondorsey we should decide if this text should be in the class documentation.  I'm not sure since it's a base class.  I'd vote that we keep it here for now.